src/containers/AdminCampaignList.jsx

```diff
@@ -341,14 +341,15 @@ export class AdminCampaignList extends React.Component {
       );
     }
     return (
-      <Fab
+      <Button
+        variant='contained'
         color="primary"
         {...dataTest("addCampaign")}
         style={theme.components.floatingButton}
         onClick={this.handleClickNewButton}
       >
-        <AddIcon />
-      </Fab>
+        Create New Campaign
+      </Button>
```

Comment:
This is one of my favorite updates.  Because Create New Campaign button makes a lot more sense than a random + covered in a green circle away from center view.